src/index.ts

```diff
@@ -8,12 +8,16 @@ import Checkpoint from './checkpoint';
 import config from './config.json';
 import * as writer from './writer';
 import checkpoints from './checkpoints.json';
+import { LogLevel } from './checkpoint/utils/logger';
 
 const dir = __dirname.endsWith('dist/src') ? '../' : '';
 const schemaFile = path.join(__dirname, `${dir}../src/schema.gql`);
 const schema = fs.readFileSync(schemaFile, 'utf8');
 
-const checkpoint = new Checkpoint(config, writer, schema, checkpoints);
+const checkpoint = new Checkpoint(config, writer, schema, checkpoints, {
+  logLevel: LogLevel.Info,
+  prettifyLogs: true
```

Comment:
I'm leaving this at info level for now, so that the logs will match what was there before. But ideally, this will be updated based on some environment variable. Should be done later